src/platform/android.rs

```diff
@@ -80,7 +80,10 @@ pub trait EventLoopExtAndroid {}
 impl EventLoopExtAndroid for EventLoop {}
```

Comment:
Do we need an active event loop or can we implement it on the nonactive one as well? (I don't remember if that's deprecated)

Reply:
I've added to event loop as well, since there's nothing wrong  with it.